zoo-css-challenge/style.css

```diff
@@ -189,7 +261,46 @@ footer {
 }
 
 /*Programs*/
+.programs {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  /* justify-content: ; */
+}
+
+.programs h2 {
+  grid-column: 1/4;
+  text-align: center;
+}
 
 .programs .card {
-  display: block;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+ 
+}
+
+.programs h3 {
+  grid-column: 1/4;;
+}
+
+.favourite {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-content: space-between;
 }
+.favourite img {
+  width: 400px;
+  grid-column: 2/3;
+  border: 5px solid green;
+}
+
+.favourite h2 {
+  grid-column: 1/3;
+  text-align: center;
+}
+
+.favourite p {
+  grid-column: 1/2;
+  text-align: center;
+  margin-top: 50px;
+}
```

Comment:
excellent effort, you've completed all the steps and effectively used Grid for responsiveness.